admin/app/components/solidus_admin/users/store_credits/index/component.rb

```diff
@@ -0,0 +1,106 @@
+# frozen_string_literal: true
+
+class SolidusAdmin::Users::StoreCredits::Index::Component < SolidusAdmin::BaseComponent
+  include SolidusAdmin::Layout::PageHelpers
+
+  def initialize(user:, store_credits:)
+    @user = user
+    @store_credits = store_credits
+  end
+
+  def model_class
+    Spree::StoreCredit
+  end
+
+  def tabs
```

Comment:
This could probably be extracted into a UserTabs module, so we do not have to add it to every user index component, no?

Reply:
I tried that early on, but had some issues when using a dynamic value for `current` (eg: `current: action_name == "edit"`. It would work on initial load but then would sometimes be a bit strange/incorrect in its display during certain flows/partial page reload. Once I have the rest of the store credits stuff out of the way I'd be happy to take another crack at it though!